<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit_console">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>621</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_products">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>480</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>商品操作</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_malluser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>用户操作（商城）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_contorluser">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>用户操作（后台）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cmddispatch">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>440</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>指令下发</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_smtpinfo">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>560</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>重新设置SMTP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_appeal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>投诉处理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_firmware">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>560</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>固件包更新</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_order">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>480</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>订单查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cmdquery">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>指令查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_webUI">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>网页引导语设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_devlist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>设备列表</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HTTP :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_edit_http">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MQTT_S :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_edit_mqttser">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListView" name="listView_ipclick">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mqtt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>MQTT配置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MQTT_C:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_edit_mqttsubcrib">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_edit_systime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>17点47分</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SMTP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_edit_smtp">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ip">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_openmqtt">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开启</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_mqtport">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>mqtt端口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>http端口</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_httpport">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>服务如果发生重启后至少等待30秒设备才能全部到场</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
